bindings/py/cpp_src/CMakeLists.txt

```diff
@@ -40,31 +40,15 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 set(CMAKE_VERBOSE_MAKEFILE OFF) # toggle for cmake debug
 message(STATUS "Configuring Python interface")
 
-#### TODO: do we need this?
-#	if (${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU" AND "${PLATFORM}" STREQUAL "linux")
-		# NOTE When building manylinux python extensions, we want the static
-		# libstdc++ due to differences in c++ ABI between the older toolchain in the
-		# manylinux Docker image and libstdc++ in newer linux distros that is
-		# compiled with the c++11 ABI. for example, with shared libstdc++, the
-		# manylinux-built extension is unable to catch std::ios::failure exception
-		# raised by the shared libstdc++.so while running on Ubuntu 16.04.
-#		set(stdlib_cxx "${stdlib_cxx} -static-libstdc++")
```

Comment:
do you think this is not a valid issue? It bundles libstdc++ with the library. Benefit would be, that if we compile the binary lib on a modern computer with c++17, we can distribute binary and people don't need boost...

Reply:
>  It bundles libstdc++ with the library. 

Ok, makes since.

Reply:
closing, correct to remove here. Already in CommonCompilerConfig